src/components/Input/Toggle/Toggle.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import TOGGLE_SIZES from "./constants";
+import {
+  ActiveArea,
+  ReactToggle,
+  ReactToggleTrack,
+  ReactToggleThumb,
+  ReactThumbCenteringContainer
+} from "./Toggle.styles";
+
+export default class Toggle extends Component {
+  handleClick = e => {
+    e.preventDefault();
+    const { onClick, onToggle, value, disabled } = this.props;
+
+    if (disabled) {
+      return;
+    }
+    onClick(e);
+    onToggle(value);
+    this.input.focus();
+  };
+
+  handleKeyPress = e => {
+    const { onToggle, value, onKeyPress } = this.props;
+    if (onKeyPress) {
+      onKeyPress(e);
+      this.input.focus();
+    } else {
+      const SPACE_CHAR_CODE = 32;
+
+      if (e.charCode === SPACE_CHAR_CODE) {
+        e.preventDefault();
+        onToggle(value);
+        this.input.focus();
+      }
+    }
+  };
+
+  render() {
+    const {
+      value,
+      onMouseOver,
+      onMouseOut,
+      onBlur,
+      onFocus,
+      size,
+      disabled
+    } = this.props;
+
+    const variant = `${value ? "active" : "inactive"}${
+      disabled ? "Disabled" : ""
+    }`;
+
+    const toggleTrackProps = {
+      value,
+      size,
+      variant,
+      role: "checkbox",
+      innerRef: el => {
+        this.input = el;
+      },
+      ...(disabled
+        ? {}
+        : {
+            onMouseOver,
+            onMouseOut,
+            onBlur,
+            onFocus,
+            tabIndex: 0,
+            onKeyPress: this.handleKeyPress
+          })
+    };
+
+    return (
+      <ActiveArea onClick={this.handleClick} size={size} disabled={disabled}>
+        <ReactToggle>
+          <ReactToggleTrack {...toggleTrackProps} />
+          <ReactThumbCenteringContainer size={size}>
+            <ReactToggleThumb
+              size={size}
+              value={value}
+              variant={variant}
+              disabled={disabled}
+            />
+          </ReactThumbCenteringContainer>
+        </ReactToggle>
+      </ActiveArea>
+    );
+  }
+}
+
+Toggle.propTypes = {
```

Comment:
Other components are introducing proptypes as `static propTypes = { ... }` inside the class.

Reply:
Fixed.